cdap-ui/app/cdap/components/EntityListView/EntityListInfo/EntityListInfo.scss

```diff
@@ -42,6 +42,7 @@ $hovered-page-background-color: rgba(219, 219, 219, 1);
       font-size: 12px;
       display: inline-block;
       vertical-align: top;
+      transform: translateY(3px);
```

Comment:
Tried several other alternatives, all of which broke either Chrome's or Firefox's alignment